features/cellular/TARGET_GENERIC_MODEM/generic_modem_driver/mbed_lib.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "ref-cell-drv", 
+		"config": {
+			"baud-rate": 115200,
+			"apn-lookup": false,
+			"at-parser-buffer-size": 256,
+			"at-parser-timeout": 8000
+	},
+	"target_overrides": {
```

Comment:
This is just an example implementation which happens to work on all modems following AT 27.00 command set. There is no dependency over COMMON_PAL. We can remove this section from this mbed_lib.json. Code base under the hood is designed to handle this, i.e., if you do not include COMMON_PAL.